### PR TITLE
Implemented visibility map in dashboard: Alex

### LINKED ISSUE #275: Integrate visibility maps into dashboard
As an astronomer, I would like to know if the stars will be visible so that I know whether its worth going out and stargazing

- Integrate visibillity data into dashboard

### PR DESCRIPTION
**Pull request by AlexOBarnes**
This closes #275 

- Integrated visibility map into dashboard

### CODE REVIEW

File: dashboard/dashboard.py [hunk 1]
```diff
@@ -14,7 +14,7 @@
 from phonenumbers import NumberParseException
 from load_dashboard_data import connect_to_db, load_from_starwatch_rds, load_forecasts_by_county_name, load_celestial_body_information
 from nasa_pipeline import nasa_pipeline, get_image_of_the_day, get_iss_location, get_moon_phase
-from aurora_map import create_aurora_map, create_visibility_map
+from aurora_map import create_aurora_map, create_visibility_map, create_body_visibility_map
```
Comment (CalebMathers) [inline dashboard/dashboard.py:17]: These functions names are great

File: dashboard/dashboard.py [hunk 2]
```diff
@@ -468,7 +468,12 @@ def get_current_datetime() -> datetime:
     selected_body = st.selectbox("Select body", bodies)
     col1, col2 = st.columns(2)
     with col1:
-       ...
+        st.header(f'{selected_body} Visibility Across The UK')
+        body_visibility = create_body_visibility_map(selected_body)
+        if body_visibility:
+           st.pyplot(body_visibility)
+        else:
+           st.write('Body map not available right now. Sorry for the inconvenience')
```
Comment (CalebMathers) [inline dashboard/dashboard.py:476]: Good integration, displays well

**Review by CalebMathers — APPROVED** [review]

File: dashboard/dashboard.py (hunk 1 above)
Comment (MurasatoNaoya) [reply dashboard/dashboard.py:17]: Yeah!  Sure do man!

File: dashboard/dashboard.py [hunk 3]
```diff
@@ -468,7 +468,12 @@ def get_current_datetime() -> datetime:
     selected_body = st.selectbox("Select body", bodies)
     col1, col2 = st.columns(2)
     with col1:
-       ...
+        st.header(f'{selected_body} Visibility Across The UK')
```
Comment (MurasatoNaoya) [inline dashboard/dashboard.py:471]: Nice to have Caleb's visibility map integrated into the dashboard, important to have it next to the the aurora visibility map.

**Review by MurasatoNaoya — APPROVED** [review]
LGTM.

File: dashboard/dashboard.py (hunk 2 above)
Comment (MurasatoNaoya) [reply dashboard/dashboard.py:476]: Agreed.

**Pull request merged**